### DISCUSSION TITLE
@parcel/core: Failed to resolve 'react-router/dom' from  './node_modules/react-router-dom/dist/index.mjs'

### DISCUSSION THREAD

**Discussion by Rajan224-glitch** [parcel-bundler/parcel#10024] [General]
please resolve this issue

**Comment by ikratkiy** [comment]
Just remove react-router-dom package and install react-router

**Reply by i-am-wiz** [reply]
I removed node_modules and then ran npm install but still didn't work. I have the following dependencies
```
"dependencies": {
        "react": "^19.0.0",
        "react-dom": "^19.0.0",
        "react-router": "^7.1.1"
}
```

**Reply by Divyeshpatil2001** [reply]
npm install react-router-dom@6.22.0   (install this router version it will work )
rm -rf .parcel-cache
npm start

**Comment by Divyeshpatil2001** [comment]
npm install react-router-dom@6.22.0 (install this router version it will work )
rm -rf .parcel-cache
npm start

**Reply by ritukumawat0** [reply]
Thankyou

**Reply by austinabrahama** [reply]
Thank you. It is working now

**Reply by Saycuso** [reply]
Bro you saved my life, thanks man.

**Reply by surya98901** [reply]
was facing same issues, Thanks. but why are we getting this error in or above version 7.1.1 any reasones ?

**Reply by Saycuso** [reply]
Oh btw, try to update it to the latest one as it's working for me now

**Reply by karthikfron** [reply]
Thank you it works for me

**Reply by sumitv** [reply]
Thank you. react-router-dom@6.22.0 version is working for me.

**Reply by sahalsemeer** [reply]
> npm install react-router-dom@6.22.0 (install this router version it will work ) rm -rf .parcel-cache npm start

why latest version doesn't work? why i want to downgrade the version?

**Comment by devongovett** [comment]
Maybe try enabling package exports? https://parceljs.org/features/dependency-resolution/#enabling-package-exports

**Reply by Kapelianovych** [reply]
Can 
```
"@parcel/resolver-default": {
  "packageExports": true
}
```
be set in a _resolver_ plugin?

**Comment by ChristopherJTrent** [comment]
This is an issue with the build cache. You can delete the cache manually like other commenters have suggested, or you can pass the `--no-cache` flag to parcel to get it to actually build your files (at least once, you'll likely have to manually restart the server every time you add a new dependency to a file)

**Comment by RichardJECooke** [comment]
The nocache flag doesn't work.  Only downgrading to react-router-dom@6.7.0 works.
